@xmatthias This is ready for review. Non-urgent change, of course.

I did have a weird test failure that seems unrelated to my PR, if you'd like to offer advice on it, shown below. I'm thinking it will pass here in the GHA, though. Probably just something weird with my setup since I'm on Windows and not using Docker, so that happens from time to time.

```
FAILED tests/test_talib.py::test_talib_bollingerbands_near_zero_values - assert 1.25e-07 != 1.25e-07
```

Appreciate you making and maintaining this great piece of FOSS!

--- a/docs/lookahead-analysis.md
+++ b/docs/lookahead-analysis.md
@@ -101,3 +101,4 @@ This could lead to a false-negative (the strategy will then be reported as non-b
 - `lookahead-analysis` has access to everything that backtesting has too.
 Please don't provoke any configs like enabling position stacking.
 If you decide to do so, then make doubly sure that you won't ever run out of `max_open_trades` amount and neither leftover money in your wallet.
+- `biased_indicators` will falsely flag FreqAI target indicators defined in `set_freqai_targets()` as biased. These are not biased and can safely be ignored.

--- a/freqtrade/optimize/analysis/lookahead_helpers.py
+++ b/freqtrade/optimize/analysis/lookahead_helpers.py
@@ -1,7 +1,7 @@
 import logging
 import time
 from pathlib import Path
-from typing import Any, Dict, List
+from typing import Any, Dict, List, Union
 
 import pandas as pd
 from rich.text import Text
@@ -21,7 +21,7 @@ class LookaheadAnalysisSubFunctions:
     def text_table_lookahead_analysis_instances(
         config: Dict[str, Any],
         lookahead_instances: List[LookaheadAnalysis],
-        caption: str | None = None,
+        caption: Union[str, None] = None,
     ):
         headers = [
             "filename",
@@ -246,7 +246,7 @@ class LookaheadAnalysisSubFunctions:
 
         # report the results
         if lookaheadAnalysis_instances:
-            caption: str | None = None
+            caption: Union[str, None] = None
             if any([
                 any([
                     indicator.startswith("&")

--- a/tests/optimize/test_lookahead_analysis.py
+++ b/tests/optimize/test_lookahead_analysis.py
@@ -13,6 +13,12 @@ from freqtrade.optimize.analysis.lookahead_helpers import LookaheadAnalysisSubFu
 from tests.conftest import EXMS, get_args, log_has_re, patch_exchange
 
 
+IGNORE_BIASED_INDICATORS_CAPTION = (
+    "Any indicators in 'biased_indicators' which are used within "
+    "set_freqai_targets() can be ignored."
+)
+
+
 @pytest.fixture
 def lookahead_conf(default_conf_usdt, tmp_path):
     default_conf_usdt["user_data_dir"] = tmp_path
@@ -138,18 +144,15 @@ def test_lookahead_helper_start(lookahead_conf, mocker) -> None:
     [
         (
             ["&indicator1", "indicator2"],
-            "Any indicators in 'biased_indicators' which are used "
-            "within set_freqai_targets() can be ignored."
+            IGNORE_BIASED_INDICATORS_CAPTION,
         ),
         (
             ["indicator1", "&indicator2"],
-            "Any indicators in 'biased_indicators' which are used "
-            "within set_freqai_targets() can be ignored."
+            IGNORE_BIASED_INDICATORS_CAPTION,
         ),
         (
             ["&indicator1", "&indicator2"],
-            "Any indicators in 'biased_indicators' which are used "
-            "within set_freqai_targets() can be ignored."
+            IGNORE_BIASED_INDICATORS_CAPTION,
         ),
         (
             ["indicator1", "indicator2"],
@@ -158,7 +161,7 @@ def test_lookahead_helper_start(lookahead_conf, mocker) -> None:
         (
             [],
             None
-        )
+        ),
     ],
     ids=(
         "First of two biased indicators starts with '&'",
@@ -166,7 +169,7 @@ def test_lookahead_helper_start(lookahead_conf, mocker) -> None:
         "Both biased indicators start with '&'",
         "No biased indicators start with '&'",
         "Empty biased indicators list",
-    )
+    ),
 )
 def test_lookahead_helper_start__caption_based_on_indicators(
     indicators,
@@ -278,7 +281,7 @@ def test_lookahead_helper_text_table_lookahead_analysis_instances(lookahead_conf
         "Pass non-empty string",
         "Pass None",
         "Don't pass caption",
-    )
+    ),
 )
 def test_lookahead_helper_text_table_lookahead_analysis_instances__caption(
     caption,

